apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,32 @@ export const Customization: Story = {
         );
     }
 }
+
+export const Formats: Story = {
+    args: {
+        width: 500,
```

Comment:
I assume we don't want to demonstrates `width` & `height` properties in this demo. It demonstrates in other stories.